src/constants/networks/avax-testnet.ts

```diff
@@ -15,11 +15,11 @@ const avaxTestnetInfo: NetworkInfo = {
   name: 'Avalanche Testnet',
   icon: AVAX,
   classicClient: createClient('https://api.thegraph.com/subgraphs/name/ducquangkstn/dmm-exchange-fuij'),
-  elasticClient: createClient('https://api.thegraph.com/subgraphs/name/kybernetwork/kyberswap-elastic-avalanche'),
+  elasticClient: createClient('https://api.thegraph.com/subgraphs/name/viet-nv/elastic-fuji'),
```

Comment:
It's your personal subgraph. Is it intentional?

Reply:
just for testing. avax testnet, no need to use company account.